test/test_clicommand.py

```diff
@@ -132,3 +133,23 @@ def test_bad_range_options(self):
                            "profilename", "hosts",
                            "a:d:b:s", "--auths",
                            "auth_1", "auth2"])
+
+    def test_a_auth_add(self):
```

Comment:
I think it would be better to combine the add and clear steps into single tests rather than depending on the order the tests are run in. But up to you.